src/core/authentication/ory.api.strategy.ts

```diff
@@ -40,12 +40,16 @@ export class OryApiStrategy extends PassportStrategy(
 
```

Comment:
I don't believe it's necessary to refresh sessions for non-interactive (API) flows.